azext_iot/central/params.py

```diff
@@ -53,7 +53,7 @@ def load_central_arguments(self, _):
         )
         context.argument(
             "device_template_id",
-            options_list=["--device-template-id"],
+            options_list=["--device-template-id", "--dt"],
```

Comment:
If this is analogous to the "Digital Twin Model Identifier" then `--dtmi` is used elsewhere but I realize the nomenclature differences. 

`--dt` does not feel optimal abbreviation because the term is so overloaded with the Digital Twins service.

Reply:
We discussed offline to opt for `dtid`.